website/docusaurus.config.js

```diff
@@ -55,8 +55,8 @@ console.log("DEBUG: ALGOLIA_INDEX_NAME = ", ALGOLIA_INDEX_NAME);
 module.exports = {
   baseUrl: '/',
   favicon: '/img/favicon.ico',
-  tagline: 'Your entire analytics engineering workflow',
-  title: 'docs.getdbt.com',
+  tagline: 'End user documentation, guides and technical reference for dbt (data build tool)',
```

Comment:
Do we even want `(data build tool)` here? We've stopped treating dbt as an initialism for this. (It still hangs out in a lot of old places, though)

Reply:
Good question. I think this tagline only shows up for SEO purposes and is a _suggestion_ for Google. I figured having the word data in there would be helpful but no strong opinion one way or the other.

Reply:
Ah that makes sense. SEO benefit probably outweighs pedantry